packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -0,0 +1,160 @@
+# Description
+
+This package contains code enabling the production and consumption of typed telemetry for Fluid Framework applications. The typed telemetry from this package is used as the backbone for different Fluid Framework cloud offerings such as dashboards and alarms for Fluid applications. This package can also be used as a reference for customizing and creating their own telemetry solution if desired.
+
+### Telemetry Destinations
```

Comment:
I think the second example below covers everything we need to say about App Insights, so I'd suggest we remove this section and "What is Application Insights" under it (just take the link to the main AppInsights docs and stick it in the example 2 section wherever we mention AppInsights for the first time).

Reply:
Updated.